model/fn-execution/src/main/resources/org/apache/beam/model/fnexecution/v1/standard_coders.yaml

```diff
@@ -455,6 +455,17 @@ examples:
 
 ---
 
+coder:
+  urn: "beam:coder:row:v1"
```

Comment:
can you double check this is not a typo.  For example with double this is:
beam:coder:double:v1

I would suspect this should be
beam:coder:float:v1

Reply:
This is a reference to row coder: https://github.com/apache/beam/blob/3173b503beaf30c4d32a4a39c709fd81e8161907/model/pipeline/src/main/proto/org/apache/beam/model/pipeline/v1/beam_runner_api.proto#L1062

Row Coder is a complex coder that is parameterized by a Beam Schema. It encodes different types of values by deferring to other coders, like `beam:coder:double:v1` for DOUBLE